raw_radar_reader/README.md

```diff
@@ -0,0 +1,72 @@
+# Raw Radar Interface
+
+This directory contains tools for interacting with raw radar data. Zendar
+makes available raw ADC samples from our FMCW radars. The example tools
+in this directory allow researchers to easily use this data format.
+
+o## Dependencies
+```
+libprotobuf
+protobuf-compiler
+```
+
+Python packages:
+```
+numpy
+matplotlib
+gzip
+os
+struct
+protobuf
+argparse
+```
+
+## Setup
+
+To use these tools, perform the following steps:
+```
+cd ./protocol
```

Comment:
can we create these protos the same way as we create the protos for the viewer?

Reply:
Isn't the way we create protos for the viewer to embed the deb file? I'd love to do that once it's ready and get rid of the files + compile step.